### DISCUSSION TITLE
Move asdf-spark to asdf-community

### DISCUSSION THREAD

**Discussion by jeffryang24** [asdf-community/.github#54] [General]
Hi @smorimoto or @vic, I like to move my [asdf-spark](https://github.com/jeffryang24/asdf-spark) repository into this organization. Could you tell me what should I do to complete this repository transfer? Thank you. :bow:

**Comment by jeffryang24** [comment]
~~After the transfer complete, then I will register the plugin shortname inside https://github.com/asdf-vm/asdf-plugins repository~~. Just checked that https://github.com/asdf-vm/asdf-plugins/blob/master/plugins/spark was registered before. Probably, this plugin will only reside in asdf-community until asdf support multichoice when installing using plugin shortname, so we can decide which repository should we use when installing using shortname strategy :laughing:

```shell
$ asdf plugin add spark
asdf detected multiple plugin sources.
1. joshuaballoch (https://github.com/joshuaballoch/asdf-spark.git)
2. jeffryang24 (https://github.com/jeffryang24/asdf-spark.git)
Please select one source (Default: 1) => [stdin]
```

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)

**Reply by jeffryang24** [reply]
Ok, noted, I will follow the guide from above discussion. Thank you :bow:
